File: app-original/plugin/anti-jail/ios/Plugin/AntiJailPlugin.swift
import Foundation
import Capacitor

/**
 * Please read the Capacitor iOS Plugin Development Guide
 * here: https://capacitorjs.com/docs/plugins/ios
 */
@objc(AntiJailPlugin)
public class AntiJailPlugin: CAPPlugin {

    @objc func validate(_ call: CAPPluginCall) {

        let jailbreakStatus = IOSSecuritySuite.amIJailbrokenWithFailMessage()
        call.resolve([
            "status": jailbreakStatus.jailbroken,
            "detalle": jailbreakStatus.failMessage
        ])
    }
}

File: app-original/plugin/anti-jail/android/src/main/java/cl/habitat/AntiJailPlugin.java
package cl.habitat;

import com.getcapacitor.JSObject;
import com.getcapacitor.Plugin;
import com.getcapacitor.PluginCall;
import com.getcapacitor.PluginMethod;
import com.getcapacitor.annotation.CapacitorPlugin;

@CapacitorPlugin(name = "AntiJail")
public class AntiJailPlugin extends Plugin {

    @PluginMethod
    public void validate(PluginCall call) {
        JSObject ret = new JSObject();
        ret.put("status", true);
        ret.put("detalle", true);
        call.resolve(ret);
    }
}
